lib/plugins/simpleRetry.js

```diff
@@ -17,30 +21,70 @@ var simpleRetry = function(worker, func, queue, job, args, options){
     self.queueObject = self.worker;
   }
 
-  self.sleep = 1000;
-  if(self.options.sleep){ self.sleep = self.options.sleep; }
+  if (self.args) {
+    jobHash = crypto.createHash('sha256').update(JSON.stringify(self.args)).digest('hex');
```

Comment:
Why use `sha256`?  That's one of the slower hashes.  You don't need to be secure here... md5 should be fine.